filecoin-hashers/src/poseidon_types.rs

```diff
@@ -26,6 +30,60 @@ lazy_static! {
     pub static ref POSEIDON_CONSTANTS_11: PoseidonConstants::<Fr, U11> = PoseidonConstants::new();
     pub static ref POSEIDON_MD_CONSTANTS: PoseidonConstants::<Fr, PoseidonMDArity> =
         PoseidonConstants::new();
+    pub static ref POSEIDON_CONSTANTS_2_PALLAS: PoseidonConstants::<Fp, U2> =
```

Comment:
Note: create Poseidon constants for the three hasher fields; put the constants into a lookup table keyed by `(field, arity)`